src/main/java/org/folio/circulation/domain/policy/lostitem/itemfee/ChargeableFee.java

```diff
@@ -0,0 +1,5 @@
+package org.folio.circulation.domain.policy.lostitem.itemfee;
+
+public interface ChargeableFee {
+  boolean isChargeable();
```

Comment:
Does this refer to whether the fee can be automatically charged by the system?

Reply:
In this context it means whether any fee have to be created at all either manual or automated. But in `AutomaticallyChargeableFee` it means what you asked.

Reply:
Is this sufficient for the current purposes because the only decision that actual cost fees are involved in is deciding if the loan can be closed immediately?

Reply:
Yes, correct.